src/main/java/com/example/ajaxonboardingproject/config/MemoryBeanPostProccesor.kt

```diff
@@ -28,20 +30,21 @@ class MemoryCheckBeanPostProcessor : BeanPostProcessor {
             val enhancer = Enhancer()
             enhancer.setSuperclass(originalBeanClass)
             enhancer.setCallback(MethodInterceptor { _, method, args, _ ->
-                val before = computeUsedMemory()
-                val result = method.invoke(bean, *args)
-                val after = computeUsedMemory()
-                if ((after - before) > MAX_MEMORY_USAGE) {
-                    throw ResponseStatusException(HttpStatus.BAD_REQUEST, "Memory usage is too high")
+                val byteArrayOutputStream = ByteArrayOutputStream()
```

Comment:
To ensure proper resource management, consider using Kotlin's use extension function, which automatically closes resources when they are no longer needed

Reply:
You're right, thanks for advice 😁